src/handlers/delete_client.rs

```diff
@@ -27,6 +27,10 @@ pub async fn handler(
     dbg!(&state.config.public_url);
     if !authenticate_client(headers, &state.config.public_url, |client_id| {
         if let Some(client_id) = client_id {
+            info!(
```

Comment:
@HarryET this is what we can do regarding client_id. Would this be sufficient?

Reply:
Yeah, thats good. I think there is a way with log? maybe a `?` syntax not sure but that can always be done later